### PR TITLE
0067954 adding examples all cases

### PR DESCRIPTION
**Pull request by mdoellerer-fc**

### CODE REVIEW

**Comment by FATCHIP-GmbH** [comment]
Please raise curl timeout to 3600

File: php/getOrder.php [hunk 1]
```diff
@@ -0,0 +1,109 @@
+<?php
+/**
+ * This example script gets information on products and on its related articles from the Tradebyte REST API.
```
Comment (FATCHIP-GmbH) [inline php/getOrder.php:3]: Description wrong - here we get orders not articles

File: php/getOrderMessages.php [hunk 2]
```diff
@@ -0,0 +1,83 @@
+<?php
+/**
+ * This example script gets information on products and on its related articles from the Tradebyte REST API.
```
Comment (FATCHIP-GmbH) [inline php/getOrderMessages.php:3]: wrong description for this file

File: php/getOrder.php (hunk 1 above)
Comment (mdoellerer-fc) [reply php/getOrder.php]: Corrected on 0afb26abf53e3882e3499f97dabdb5dd511c6ed2

File: php/getOrderMessages.php (hunk 2 above)
Comment (mdoellerer-fc) [reply php/getOrderMessages.php]: Corrected on 0afb26abf53e3882e3499f97dabdb5dd511c6ed2

File: php/getOrderMessages.php [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    sendReceivedFlagToTradebyte($messagesImported, $sMerchantId, $sChannelId, $sApiUser, $sApiPassword);
+
+} else {
+    print_r($sResponse);
+}
+
+function sendReceivedFlagToTradebyte($arrayOfMessageIds, $sMerchantId, $sChannelId, $sApiUser, $sApiPassword){
+    foreach ($arrayOfMessageIds as $messageId){
+
+        $sUrl = "https://rest.trade-server.net/" . $sMerchantId . "/messages/" . $messageId . "/?channel=" . $sChannelId;
```
Comment (kinimodmeyer) [inline php/getOrderMessages.php:64]: https://rest.trade-server.net/" . $sMerchantId . "/messages/" . $messageId . "/exported

Comment (Bugfighter) [reply php/getOrderMessages.php:64]: Hi Dominik,
what you write does not match the official API documentation - is this a hidden feature or do we use an old version of the docs?
POST https://rest.trade-server.net/(HNR)/messages/(MSGID)/processed?channel=(CHID)
(see https://tradebyte.io/documentation/basics-about-data-exchange/rest-calls-for-channels-and-merchants/)
Thx for your feedback!
Hendrik

Comment (kinimodmeyer) [reply php/getOrderMessages.php:64]: @Bugfighter accordingly to this link you missing "/processed" in the URL.

Comment (Bugfighter) [reply php/getOrderMessages.php:64]: Correct - so we use "processed?channel= . $sChannelId" and not "/exported".

Comment (mdoellerer-fc) [reply php/getOrderMessages.php]: Ok. Correction is in place.

**Pull request merged**